docs/upgrade/v1-4-1-to-v1-4-2.md

```diff
@@ -113,3 +113,20 @@ A possible cause is processes related to orphan engines of the Longhorn Instance
 Related issues:
   - [[BUG] v1.4.0 -> v1.4.1-rc1 upgrade stuck in Pre-drained and the node stay in Cordoned](https://github.com/harvester/harvester/issues/7366)
   - [[IMPROVEMENT] Cleanup orphaned volume runtime resources if the resources already deleted](https://github.com/longhorn/longhorn/issues/6764)
+
+### 2. High CPU usage on the Harvester
+
+After upgrading to v1.4.2, the Harvester may experience high CPU usage. This issue is caused by the `backup-target` setting. In Harvester v1.4.2, we introduced a new field `refreshIntervalInSeconds`. If it's empty or 0, the Harvester will keep refreshing the backup target, which causes high CPU usage.
+
+![](/img/v1.4/image/high-cpu-load-issue.png)
+
+To fix this, please update `refreshIntervalInSeconds` to a bigger value, like 60.
+
+```
+'{"type":"nfs","endpoint":"nfs://longhorn-test-nfs-svc.default:/opt/backupstore"}'
```

Comment:
Should we include the `"refreshIntervalInSeconds"` k/v pair in this example?

Reply:
Yes, updated it. Thanks.